containers/datalab/Dockerfile.in

```diff
@@ -36,5 +36,7 @@ ENV EXPERIMENTAL_KERNEL_GATEWAY_URL=""
 # Startup
 ENV DATALAB_VERSION _version_
 ENV DATALAB_COMMIT _commit_
-ENTRYPOINT [ "/datalab/run.sh" ]
+ENV DEPLOY_VM "false"
```

Comment:
Sounds good.

That script would be for demo purpose so I think it is Okay to be not cross platform. Users are responsible for creating the VM in their own way.